src/hook.rs

```diff
@@ -62,6 +62,34 @@ impl Hook {
     pub fn try_recv(&self) -> Result<InputEvent, std::sync::mpsc::TryRecvError> {
         InnerHook::try_recv()
     }
+
+    /// Hook::recv() should be an option besides Hook::try_recv() when you want park the current thread until there is an event to process.
```

Comment:
Please reword description so that this doc does sound like statement and does not duplicate try_recv() example.

Maybe
`This is blocking alternative to Hook::try_recv(). That is it will block the current thread until there is an event from the low-level hook(s) running in the background thread(s), and then will return it.`